apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -43,7 +43,7 @@ type ModelManagerV4 interface {
 	CreateModel(args params.ModelCreateArgs) (params.ModelInfo, error)
 	DumpModels(args params.DumpModelRequest) params.StringResults
 	DumpModelsDB(args params.Entities) params.MapResults
-	ListModelsWithInfo(user params.Entity) (params.ModelInfoResults, error)
+	ListModelsWithInfo(user params.Entity) (params.ModelSummaryResults, error)
```

Comment:
I think we should change the API name to ListModelsWithSummary or ListModelSummaries.